.dassie/app/models/collection.rb

```diff
@@ -4,5 +4,5 @@ class Collection < ActiveFedora::Base
   include ::Hyrax::CollectionBehavior
   # You can replace these metadata if they're not suitable
   include Hyrax::BasicMetadata
-  self.indexer = Hyrax::CollectionWithBasicMetadataIndexer
+  self.indexer = Hyrax::CollectionWithBasicMetadataIndexer 
```

Comment:
I wonder if you ran rubocop.  I would have expected this change to fail with the space at the end of the line.  Maybe rubocop doesn't run over the .dassie directory.

Reply:
@elrayle rubocop is set up in main to specifically not run on .dassie